django/core/paginator.py

```diff
@@ -52,6 +53,9 @@ def page(self, number):
         top = bottom + self.per_page
         if top + self.orphans >= self.count:
             top = self.count
+
+        if hasattr(self.object_list, 'ordered') and not self.object_list.ordered:
+            warnings.warn("Inconsistent pagination when no ordering is specified", RuntimeWarning)
```

Comment:
I makes it easier to filter them (e.g. `warnings.simplefilter('error', DjangoRuntimeWarning)`) during a test suite execution of for applications wrapping around Django.

I initially suggested using them because I was reviewing the cache code for a PR and noticed the usage of `CacheKeyWarning(DjangoRuntimeWarning)`. Now that I grep for `RuntimeWarning` through the code base I realize it's the only use of `DjangoRuntimeWarning`.

I believe it could be useful to use `DjangoRuntimeWarning` everywhere but as `CacheKeyWarning` is its only current usage it might be worth questioning its existence.